eng/build.ps1

```diff
@@ -93,8 +92,7 @@ function Print-Usage() {
   Write-Host "  -help                     Print help and exit"
   Write-Host ""
   Write-Host "Test actions"
-  Write-Host "  -test32                   Run unit tests in the 32-bit runner"
-  Write-Host "  -test64                   Run units tests in the 64-bit runner"
+  Write-Host "  -testArch                 Maps to --arch parameter of dotnet test"
```

Comment:
Consider documenting the accepted values

Reply:
I wanted to leave that out because the set of values isn't constant. It's whatever the underlying `dotnet test` supports and that changes across versions.